apic_ml2/neutron/services/l3_router/l3_apic.py

```diff
@@ -266,50 +120,24 @@ def remove_router_interface(self, context, router_id, interface_info):
     def create_floatingip(self, context, floatingip):
         res = super(ApicL3ServicePlugin, self).create_floatingip(
             context, floatingip)
-        port_id = floatingip.get('floatingip', {}).get('port_id')
-        self._notify_port_update(port_id)
-        if res:
-            res['status'] = self._update_floatingip_status(context, res['id'])
+        fip_context = api.FipContext(res, original=floatingip)
```

Comment:
I guess I was trying to remain true to the original implementation;

https://github.com/noironetworks/apic-ml2-driver/blob/master/apic_ml2/neutron/services/l3_router/l3_apic.py#L266-L273

It looks like they get the port_id from the floatingip object used for the create, but we also need the actual floating IP object returned from the superclass so that we can update its status in the postcommit call. I guess that's why I used current and original this way. Maybe I should create different member names instead for this, in order to make it less confusing to those familiar with the ML2 convention?